tests/load/pipeline/test_merge_disposition.py

```diff
@@ -597,39 +731,53 @@ def duplicates_no_child():
 
 @pytest.mark.parametrize(
     "destination_config",
-    destinations_configs(default_sql_configs=True, supports_merge=True),
+    destinations_configs(
+        default_sql_configs=True, local_filesystem_configs=True, supports_merge=True
+    ),
     ids=lambda x: x.name,
 )
 @pytest.mark.parametrize("merge_strategy", ("delete-insert", "upsert"))
 def test_complex_column_missing(
     destination_config: DestinationTestConfiguration,
     merge_strategy: TLoaderMergeStrategy,
 ) -> None:
+    if destination_config.table_format == "delta":
+        pytest.skip(
+            "Record updates that involve removing elements from a complex"
```

Comment:
In the SQL implementation for `upsert` we use subqueries for this.

`delta-rs` doesn't currently support subqueries in predicates (see this [ticket](https://github.com/delta-io/delta-rs/issues/2668) that I created on their repo).